pkg/configuration/configuration.go

```diff
@@ -48,6 +48,20 @@ type AcceptedKernelTaintsInfo struct {
 	Module string `yaml:"module" json:"module"`
 }
 
+// SkipScalingTestDeploymentNamesInfo contains a list of names of deployments that should be skipped by the scaling tests to prevent issues
+type SkipScalingTestDeploymentNamesInfo struct {
+
+	// Deployment name that can be skipped by the scaling tests
+	Name string `yaml:"name" json:"name"`
```

Comment:
Shouldn't we allow here to specify the namespace as well?

Reply:
Good idea, I will add a namespace field.